sdk/react/panels/MoreOptionScreen.js

```diff
@@ -180,9 +187,10 @@ class MoreOptionScreen extends React.Component {
         buttonIcon = this.props.config.icons.expand;
         break;
       case BUTTON_NAMES.PLAYBACK_SPEED:
+        const fontStr = this.props.selectedPlaybackSpeedRate;
```

Comment:
in the future, should we use full names for properties? 
For example: `defaultFont` instead of `fontStr`